app/src/main/res/layout/fragment_crime.xml

```diff
@@ -1,52 +1,109 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
+    android:id="@+id/linearLayout2"
```

Comment:
just FYI - you have a `ConstraintLayout` as root, yet you set the id to `linearLyout`. Not a critical thing if you can manage this, although it is an unnecessary added cognitive load. To keep things clean, you could apply consistent naming here, i.e `constraintLayout`, or simply id it as `root`, since its the root layout of that view